README.md

```diff
@@ -18,7 +18,18 @@ once the Diocesan Calendar is defined, it can be requested using the `diocesanCa
 
 The National and Diocesan Calendar data can be defined directly through the interfaces offered by this frontend.
 
-# Localization of the Frontend
+## Development
+
+The HTML is plain PHP. (It could be served locally with `php -S localhost:8000`.) (You'll need php-intl installed.)
```

Comment:
I would rephrase this something along the lines of "The pages consist of plain HTML and PHP, without any build steps" if that's what you're trying to convey

Reply:
Perhaps we should also mention the minimum required PHP version? Currently it is PHP 7.4